src/AutoRest.CSharp/MgmtTest/Extensions/CodeWriterExtensions.cs

```diff
@@ -33,16 +34,16 @@ internal static class CodeWriterExtensions
         /// <param name="type"></param>
         /// <param name="includeCollectionInitialization"></param>
         /// <returns></returns>
-        public static CodeWriter AppendExampleValue(this CodeWriter writer, ExampleValue exampleValue, CSharpType? type = null, bool includeCollectionInitialization = true)
+        public static CodeWriter AppendExampleValue(this CodeWriter writer, InputExampleValue exampleValue, CSharpType? type = null, bool includeCollectionInitialization = true)
```

Comment:
since we have the type unified, maybe we could try to remove this class and replace it by this class: https://github.com/Azure/autorest.csharp/blob/f823fd228ed51588df6703d8b35d00d0764818a6/src/AutoRest.CSharp/LowLevel/Extensions/Snippets.ExampleValues.cs#L26
which is designed to be a replacement of this
But this could happen later since we already have a really big PR

Reply:
please add a TODO here if we plan this replacement to happen later

Reply:
Created https://github.com/Azure/autorest.csharp/issues/4717 for tracking